charts/prefect-worker/templates/deployment.yaml

```diff
@@ -67,7 +67,7 @@ spec:
             - --type
             - {{ .Values.worker.config.type | quote }}
             - --pool
-            - {{ .Values.worker.config.workPool | quote }}
+            -  {{ required "A Work Pool Name is required (worker.config.workPool)" .Values.worker.config.workPool | quote }}
```

Comment:
i can never tell - do we need a leading space hyphen thing here?

`{{- required`

Reply:
it through some validation error when i had that in. good q tho!